package.json

```diff
@@ -26,10 +26,12 @@
   },
   "dependencies": {
     "apollo-server": "2.9.3",
+    "apollo-server-express": "^2.9.7",
```

Comment:
Do we still need the `apollo-server` package? Can we test it and see if it works also without it?

Reply:
I forgot to remove :sweat_smile: